package.json

```diff
@@ -84,7 +84,8 @@
     "typedoc": "^0.21.5",
     "typedoc-plugin-markdown": "^3.10.4",
     "typescript": "^4.2.2",
-    "yargs": "^17.3.1"
+    "yargs": "^17.3.1",
+    "ts-essentials": "^9.3.0"
```

Comment:
is this package needed?

Reply:
thanks! can you please undo the yarn.lock changes too?

Reply:
I did remove the yarn.lock file followed w a yarn install to create a new one
That should do it right?

Reply:
i think the best way is to do the following:
```bash
git pull
git merge origin/main
```
then resolve the merge conflicts on the files using your IDE
for `yarn.lock`, accept all incoming changes for the whole file then save
stage the changes
complete the merge commit
`git push`